homeassistant/components/seitron/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""The seitron integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+import async_timeout
+from pyseitron.seitron_gateway import SeitronGateway
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client, config_entry_oauth2_flow
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from . import api
+from .const import DOMAIN  # seitron
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.CLIMATE]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up seitron from a config entry."""
+    _LOGGER.info("Init: setup: EntryId %s", entry.entry_id)
```

Comment:
please remove this and all other info logging. This is not needed to be printed as info to the user.

Reply:
fixed